frontend/src/metabase/admin/people/const.js

```diff
@@ -0,0 +1,4 @@
+export const USER_STATUS = {
+  active: "active",
+  deactivated: "deactivated",
+};
```

Comment:
Other files with constants are called `constants`. Should we use the same name here, maybe?